x-pack/legacy/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/beats/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,307 @@
+                        <FormattedMessage
+                          id="xpack.monitoring.metricbeatMigration.beatsInstructions.enableMetricbeatModuleHttpEnabledDirectionsLinkText"
+                          defaultMessage="enable an HTTP endpoint for the {beatType} instance being monitored"
+                          values={{
+                            beatType
+                          }}
+                        />
+                      </EuiLink>
+                    ),
+                    beatType: beatType || UNDETECTED_BEAT_TYPE
+                  }}
+                />
+              </p>
+            </EuiText>
+          )}
+        />
+        {securitySetup}
+      </Fragment>
+    )
+  };
+
+  const configureMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.beatsInstructions.configureMetricbeatTitle', {
+      defaultMessage: 'Configure Metricbeat to send to the monitoring cluster'
+    }),
+    children: (
+      <Fragment>
+        <EuiText>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.beatsInstructions.configureMetricbeatDescription"
+            defaultMessage="Make these changes in your {file}."
+            values={{
+              file: (
+                <Monospace>metricbeat.yml</Monospace>
+              )
+            }}
+          />
+        </EuiText>
+        <EuiSpacer size="s"/>
+        <EuiCodeBlock
+          isCopyable
+        >
+          {`output.elasticsearch:
+  hosts: ["${esMonitoringUrl}"] ## Monitoring cluster
+
+  # Optional protocol and basic auth credentials.
+  #protocol: "https"
+  #username: "elastic"
+  #password: "changeme"
+`}
+        </EuiCodeBlock>
+        {securitySetup}
+      </Fragment>
+
+    )
+  };
+
+  const startMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.beatsInstructions.startMetricbeatTitle', {
+      defaultMessage: 'Start Metricbeat'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <EuiLink
+            href={`${ELASTIC_WEBSITE_URL}guide/en/beats/metricbeat/${DOC_LINK_VERSION}/metricbeat-starting.html`}
+            target="_blank"
+          >
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.beatsInstructions.startMetricbeatLinkText"
+              defaultMessage="Follow the instructions here"
+            />
+          </EuiLink>
+        </p>
+      </EuiText>
+    )
+  };
+
+  let migrationStatusStep = null;
+  if (product.isInternalCollector || product.isNetNewUser) {
+    let status = null;
+    if (hasCheckedStatus) {
+      status = (
+        <Fragment>
+          <EuiSpacer size="m"/>
+          <EuiCallOut
+            size="s"
+            color="warning"
+            title={i18n.translate('xpack.monitoring.metricbeatMigration.beatsInstructions.isInternalCollectorStatusTitle', {
+              defaultMessage: `We have not detected any monitoring data coming from Metricbeat for this Beats.
```

Comment:
Should `Beats` maybe be singular here `Beat`? or if plural maybe something like: `...from these Beats`?